src/services/github.js

```diff
@@ -0,0 +1,5 @@
+export const fetchUser = () => {
+  return fetch('https://api.github.com/users/dorjepadma')
```

Comment:
instead of dorjepadma you'll want to pass through whatever name is inputted in the parens in line 1, and then use that as a template literal... somewhat like this

`export const fetchUser = (name) => {
return fetch('url/${name}')
`